amundsen_application/static/js/ducks/search/types.ts

```diff
@@ -29,6 +29,16 @@ export interface SearchAllResponsePayload extends SearchResponsePayload {
   tables: TableSearchResults;
   users: UserSearchResults;
 };
+export interface InlineSearchResponsePayload {
+  tables: TableSearchResults;
+  users: UserSearchResults;
+};
+export interface InlineSearchUpdatePayload {
+  searchTerm: string;
+  selectedTab: ResourceType;
```

Comment:
Do we need the `selectedTab` here? Also I believe we have renamed this to `resource` or `resourceType` since we don't have any tabs anyways.

Reply:
We do for this case. The related action for this type is fired when a user clicks on the inline result, and when this happens we want the search state to update with the inline results data as opposed to having to fire another request to get the same exact data.  In the case where the user clicks on any of the "view all results" for users, this lets the app know that "People" should be selected on the search page.

Reply:
I think I also got confused because `selectedTab` is still used a lot of places where I copy paste things. I'll file an official "replace all references of selectedTab with resourceType" ticket if it doesn't already exist. I think it'll be best to do it all at once else the issue will keep propagating from copy-paste.